### ISSUE TITLE
Format for documenting variables and placeholders?

### ISSUE THREAD

**Issue by tang-mm** [thegooddocsproject/templates#262]
Suggestions from the [Google style guide](https://developers.google.com/style/placeholders):
- wrapping placeholders in a <var> element and use uppercase characters with underscore delimiters, such as `PROJECT_ID`
- In Markdown, wrap inline placeholders in backticks (`), and use an asterisk (*) before the first backtick and after the second one (*`PLACEHOLDER_NAME`*).
- If the context in which your placeholders appear makes using uppercase characters with underscore delimiters a bad idea, use something else that makes sense to you, but be internally consistent.

However, in our templates, curly braces `{ }` have been commonly used. We should have a set of guidelines regarding the format.

See [the related review comment](https://github.com/thegooddocsproject/templates/pull/252/files/5ab3d366861e8cabad4f4c004e0abd47247ecddb#r917303589)

**Comment by heykayla** [comment]
Agreed to continue using curly brackets for both placeholder text and writer's notes.
